alpha/veneer/basic/basic.go

```diff
@@ -0,0 +1,83 @@
+package basic
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-registry/alpha/action"
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/pkg/image"
+)
+
+type BasicVeneer struct {
```

Comment:
Nit: avoid unnecessary stuttering between the package name `basic` and the type `BasicVeneer`
```suggestion
type Veneer struct {
```

When used in other packages, this looks like `basic.Veneer`